draft-birkholz-scitt-receipts.md

```diff
@@ -301,7 +307,9 @@ The following steps must be followed to verify a Receipt:
 
         root := compute_root(LeafHash, proof)
 
-6. Verify the signature with the root hash as payload using the certificate chain established by the node certificate embedded in the receipt and the service certificate part of the service's parameters (see {{parameters}}).
+6. Extract the Issued At parameter from sign_protected of the leaf_info field of the receipt contents.
```

Comment:
What does extract mean? It sounds like removal, but that cannot be right, yes?

Reply:
I meant something in the realm of "read" or "access". Maybe the step isn't really necessary and can be inlined into the following.